src/Magnum/ImGuiIntegration/Context.cpp

```diff
@@ -324,6 +342,8 @@ void Context::drawFrame() {
                 {pcmd->ClipRect.z, fbSize.y() - pcmd->ClipRect.y}}
                     .scaled(_supersamplingRatio)});
 
+            /* Only > 0 if ImGuiBackendFlags_RendererHasVtxOffset is set */
+            _mesh.setBaseVertex(pcmd->VtxOffset);
             _mesh.setCount(pcmd->ElemCount);
             _mesh.setIndexBuffer(_indexBuffer, pcmd->IdxOffset*sizeof(ImDrawIdx),
```

Comment:
IIRC, the same effect could be achieved by having a 32-bit ImDrawIdx, right? As a fallback solution, especially since the `WEBGL_draw_instanced_base_vertex_base_instance` extension is not that widely available as far as I know.

As far as I know, it's not runtime-switchable, has to be defined in user code before including anything from imgui and thus we can't control it ourselves. I think it would make sense to have that documented at least -- for example [before the Multiple contexts section](https://github.com/mosra/magnum-integration/blob/af8b343b3466e92a7cbcdd4a53757150ea754273/src/Magnum/ImGuiIntegration/Context.h#L253).

Actually, it would be good to have it documented together with the extension requirements ("use 32-bit indices if you can't ensure the extensions are present, especially on WebGL"), and maybe mentioning that [ImPlot is a widget often pushing the limits](https://github.com/mosra/magnum-integration/issues/82#issuecomment-800179085). Thanks!